packages/website/docs/_samples/main/Input/CustomIcon/CustomIcon.md

```diff
@@ -0,0 +1,11 @@
+```html title="example.html"
+<ui5-input value="Some initial value" show-clear-icon >
+    <ui5-icon id="custom-icon" name="search" slot="icon"></ui5-icon>
+</ui5-input>
+<script type="module">
+    import inputCss from "@ui5/webcomponents/dist/styles/InputIcon.module.css";
```

Comment:
Here in the sample, we need to also switch to the new css module name and camelCase classes (Icon.module.css, inputIcon)